lunatrace/bsl/ml/python/README.md

```diff
@@ -39,17 +39,17 @@ OPENAI_API_KEY=$(aws secretsmanager get-secret-value --secret-id lunasec-OpenAIS
 ```
 
 ### Proto
-The protobuf definitions are in `lunasec/lunatrace/bsl/proto/`. To update the protobuf definitions, you will need `protoc`, `protoc-gen-twirpy`, and `protoc-gen-go`:
+The protobuf definitions are in `lunasec/lunatrace/bsl/proto/`. To update the protobuf definitions, you will need `protoc` and `protoc-gen-go`:
```

Comment:
Why aren't these in the folder with all of the python that they correspond with?  Wouldnt that make more sense than a folder called proto that ends up with a big mishmash of protobuf files in it from the whole project?

Reply:
or i guess its used by both the go code AND the python so its sort of tricky who owns it. That makes sense then